starknet_devnet/chargeable_account.py

```diff
@@ -0,0 +1,24 @@
+"""
+Account that is charged with a fee when nobody else can be charged.
+"""
+
+from starknet_devnet.account import Account
+
+
+class ChargeableAccount(Account):
+    """
+    A well-funded account that can be charged with a fee when no other account can.
+    """
+
+    PRIVATE_KEY = 0x5FB2959E3011A873A7160F5BB32B0ECE
```

Comment:
Maybe add it to development section.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/384/commits/12c8fdcf43d033795f945a61991198c5108f8e74